scripts/test_is_generated_code_in_sync.sh

```
@@ -1,7 +1,7 @@
-#!/bin/sh
-set -eu
+#!/bin/bash
+set -euo pipefail
 
-cd $(dirname $0)/../api
+cd "$(dirname $0)/../api"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)